src/Broadcast.js

```diff
@@ -28,6 +28,26 @@ class Broadcast {
     });
   }
 
+  static atExcept(excludes, source, message, wrapWidth, useColor, formatter) {
```

Comment:
I'd prefer if this keep the source, message arguments first so its prototype matches more closely the other Broadcast methods. So: `source, message, excludes`